[ts_converter] Support prim::max and prim::if with multiple outputs

SherlockNoMad opened the pull request:
Summary: As title.

Test Plan: test_converter.py

Reviewed By: angelayi

Differential Revision: D60147455

ydwu4 commented on test/export/test_converter.py:
```diff
@@ -869,6 +936,22 @@ def forward(self, x):
         #         )[0], 1
         #     )
 
```
maybe turn this into a parametrized test and merge these two tests?

SherlockNoMad replied:
hmm, I tried 

jit.script() failed at `func(x_len, y_len)`

I guess scripting doesn't support capturing callable type.

ydwu4 commented on test/export/test_converter.py:
```diff
@@ -346,6 +382,37 @@ def forward(self, x: torch.Tensor, y: torch.Tensor):
                 M()(torch.tensor(False), torch.tensor(4)),
             )
 
+    @unittest.skip('Wrong fx subgraph for cond, need to fix')
```
@jiashenC can take a look?

SherlockNoMad committed:
[ts_converter] Support prim::max and prim::if with multiple outputs (#131593)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/131593

As title.

Test Plan: test_converter.py

Reviewed By: ydwu4, angelayi

Differential Revision: D60147455
